esmvaltool/cmorizers/data/downloaders/datasets/esacci_landcover.py

```diff
@@ -0,0 +1,49 @@
+"""Script to download ESACCI-LANDCOVER pft data from the CEDA."""
+
+from datetime import datetime
+
+from dateutil import relativedelta
+
+from esmvaltool.cmorizers.data.downloaders.ftp import CCIDownloader
+
+
+def download_dataset(config, dataset, dataset_info, start_date, end_date,
+                     overwrite):
+    """Download dataset.
+
+    Parameters
+    ----------
+    config : dict
+        ESMValTool's user configuration
+    dataset : str
+        Name of the dataset
+    dataset_info : dict
+         Dataset information from the datasets.yml file
+    start_date : datetime
+        Start of the interval to download
+    end_date : datetime
+        End of the interval to download
+    overwrite : bool
+        Overwrite already downloaded files
+    """
+    if not start_date:
+        start_date = datetime(1992, 1, 1)
+    if not end_date:
+        end_date = datetime(2020, 12, 31)
+
+    loop_date = start_date
+
+    downloader = CCIDownloader(
+        config=config,
+        dataset=dataset,
+        dataset_info=dataset_info,
+        overwrite=overwrite,
+    )
+    downloader.ftp_name = 'land_cover'
+    downloader.connect()
+    downloader.set_cwd('/pft/v2.0.8/')
+    downloader.download_folder('.')
```

Comment:
I remember that @axel-lauer mentioned that this is also downloading unnecessary files. Has this been resolved now?

Reply:
I did some tests on an amended version of this one to download only the .nc files properly as discussed with @axel-lauer indeed but it is still downloading also the zip file and txt, still worning on it

Reply:
@schlunma @axel-lauer The downloader is fixed and now it's downloading only the .nc files in the /pft/v2.0.8/ folder as expected.